cli/sigstore.go

```diff
@@ -0,0 +1,174 @@
+package cli
```

Comment:
Does this best belong in cli or another package/directory? The other files in the cli package create Cobra commands

Reply:
Thats a @jpower432 decision.

Reply:
Since the methods in this file are private, I would say either leave it as it is or put it into an internal package under the `cli` package.